src/main/java/apoc/number/exact/Exact.java

```diff
@@ -0,0 +1,99 @@
+package apoc.number.exact;
+
+import apoc.Description;
+import org.apache.commons.lang.StringUtils;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.UserFunction;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.math.RoundingMode;
+
+import static java.lang.Math.pow;
+
+/**
+ * @author AgileLARUS
+ *
+ * @since 17 May 2017
+ */
+public class Exact {
+
+	@UserFunction
+	@Description("apoc.number.exact.add(stringA,stringB) | return the sum's result of two large numbers")
+	public String add(@Name("stringA")String stringA, @Name("stringB")String stringB){
+
+		return new BigDecimal(stringA).add(new BigDecimal(stringB)).toPlainString();
```

Comment:
And change description to use `-`, instead of `|` to not conflict with asciidoc-tables